test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -80,30 +80,34 @@ public class ScanConsistencyIT extends AccumuloClusterHarness {
 
   private static final Logger log = LoggerFactory.getLogger(ScanConsistencyIT.class);
 
-  public static void main(String[] args) {
-    /**
-     * @formatter:off
-     * Note: In order to run main,
-     * 1) Build the project
-     * 2) Copy the accumulo test jar (in /test/target/) into your accumulo installation's
-     * lib directory*
-     * 3) Copy the JUnit dependencies into your accumulo installation's lib directory:
-     * "mvn dependency:copy-dependencies -DincludeGroupIds="org.junit.jupiter"
-     * cp test/target/dependency/junit-jupiter-* $ACCUMULO_HOME/lib/"
-     *
-     * *Ensure the test jar is in lib before the tablet servers start. Restart tablet
-     * servers if necessary.
-     *
-     * Now, this can be run with
-     * "accumulo org.apache.accumulo.test.ScanConsistencyIT <props-file> <tmp-dir> <table> <sleep-time>"
-     *      <props-file>: An accumulo client properties file
-     *      <tmp-dir>: tmpDir field for the TestContext object
-     *      <table>: The name of the table to be created
-     *      <sleep-time>: The time to sleep (ms) after submitting the various concurrent tasks
-     * @formatter:on
-     */
+  /**
+   * Note: In order to run main,<br>
+   * 1) Build the project<br>
+   * 2) Copy the accumulo test jar (in /test/target/) into your accumulo installation's lib
+   * directory*<br>
+   * 3) Copy the JUnit dependencies into your accumulo installation's lib directory:<br>
+   * $ mvn dependency:copy-dependencies -DincludeGroupIds="org.junit.jupiter"<br>
+   * $ cp test/target/dependency/junit-jupiter-* $ACCUMULO_HOME/lib/<br>
+   * <br>
+   * *Ensure the test jar is in lib before the tablet servers start. Restart tablet servers if
+   * necessary.<br>
+   * <br>
+   * Now, this can be run with<br>
+   * $ accumulo org.apache.accumulo.test.ScanConsistencyIT [props-file] [tmp-dir] [table]
+   * [sleep-time]<br>
+   * <br>
+   *
+   * [props-file]: An accumulo client properties file<br>
+   * [tmp-dir]: tmpDir field for the TestContext object<br>
+   * [table]: The name of the table to be created<br>
+   * [sleep-time]: The time to sleep (ms) after submitting the various concurrent tasks<br>
+   * <br>
```

Comment:
I don't know if this works, but I was thinking something more along this:

```suggestion
   * Note: In order to run main,
   * <ol>
   * <li>Build the project
   * <li>Copy the accumulo test jar (in /test/target/) into your accumulo installation's lib directory
   * <li>Copy the JUnit dependencies into your accumulo installation's lib directory:
   * <ul>
   * <li>mvn dependency:copy-dependencies -DincludeGroupIds="org.junit.jupiter"
   * <li>cp test/target/dependency/junit-jupiter-* $ACCUMULO_HOME/lib/
   * </ul>
   * <li>Ensure the test jar is in lib before the tablet servers start. Restart tablet servers if
   * necessary.
   * <li>Run with: bin/accumulo org.apache.accumulo.test.ScanConsistencyIT [props-file] [tmp-dir] [table]
   * [sleep-time]
   * </ol>
   *
   * [props-file]: An accumulo client properties file<br>
   * [tmp-dir]: tmpDir field for the TestContext object<br>
   * [table]: The name of the table to be created<br>
   * [sleep-time]: The time to sleep (ms) after submitting the various concurrent tasks
```

Reply:
Reformatted similarly. From running the QA checks, maven didn't like the unordered list nested within the ordered list which is why I have the third list element the way it is.